site/pages/api/create-consequence-network.api.ts

```diff
@@ -25,13 +24,12 @@ import { getSession } from "../../utils/apiUtils/auth";
 
 const createConsequenceNetwork = async (req: NextApiRequest, res: NextApiResponse): Promise<void> => {
     try {
-        const isFromTemplate = isDisruptionFromTemplate(req);
         const { template, addAnotherConsequence } = req.query;
```

Comment:
We should remove references to template as we have moved the functionality to a separate api endpoint. 

Also addAnotherConsequence should be moved to line 32 to be: 
```const { draft, isFromTemplate, addAnotherConsequence } = req.query;```

This also applies to the other create-consequence apis.

Reply:
done